### PR TITLE
Optional Abort and Build Script

### PR DESCRIPTION
**Pull request by thehappydinoa**
- Added one of these: `DO YOU WANT TO IGNORE OUR WARNING AND RISK BOOTLOOPS?` for OOS and [Nexus users](https://forum.xda-developers.com/showpost.php?p=78567025&postcount=1251)
- Rearanged `install.sh` so all choices happen at the begining
- Make a build script in `.github/build.sh` 
- Git ignored the `builds` folder
- Disabled `DEBUG` flag that raises `cmd: Failure calling service activity: Failed transaction (2147483646)` errors

### CODE REVIEW

**Review by JohnFawkes — APPROVED** [review]

File: common/install.sh [hunk 1]
```diff
@@ -48,44 +43,42 @@ chooseold() {
...
 fi
 
-if [ $API -ge 27 ]; then
-  rm -rf $INSTALLER/system/framework
+ignorewarning() {
+  ui_print "   DO YOU WANT TO IGNORE OUR WARNING AND RISK BOOTLOOPS?"
+  ui_print "   Vol Up = Yes, Vol Down = No"
+  if $FUNCTION; then
+    ui_print " "
+    ui_print "   Ignoring warning..."
```
Comment (ThatNerdyPikachu) [inline common/install.sh]: Maybe we could use ``You have been warned!`` instead?

File: common/install.sh [hunk 2]
```diff
@@ -48,44 +43,42 @@ chooseold() {
...
+  ui_print "   Press Vol Up:"
+  $FUNCTION "UP"
+  ui_print "   Press Vol Down:"
+  $FUNCTION "DOWN"
 fi
 
-if [ $API -ge 27 ]; then
-  rm -rf $INSTALLER/system/framework
+ignorewarning() {
+  ui_print "   DO YOU WANT TO IGNORE OUR WARNING AND RISK BOOTLOOPS?"
```
Comment (ThatNerdyPikachu) [inline common/install.sh]: Warnings instead?

File: common/install.sh [hunk 3]
```diff
@@ -48,44 +43,42 @@ chooseold() {
...
-ui_print "   ! Legacy device detected! Using old keycheck method"
-ui_print " "
-ui_print " - Vol Key Programming -"
-ui_print "   Press Vol Up:"
-$FUNCTION "UP"
-ui_print "   Press Vol Down:"
-$FUNCTION "DOWN"
+if [ "$OOS" ]; then
+  ui_print " "
+  ui_print "   Pix3lify has a hard time with OnePlus devices!"
```
Comment (ThatNerdyPikachu) [inline common/install.sh]: ``Pix3lify has been known to not work and cause issues on devices running OxygenOS`` or smth

File: common/install.sh (hunk 2 above)
Comment (thehappydinoa) [reply common/install.sh]: I just moved that bit to the bottom

Comment (ThatNerdyPikachu) [reply common/install.sh]: Hm?

Comment (ThatNerdyPikachu) [reply common/install.sh]: I meant in the message.

Comment (thehappydinoa) [reply common/install.sh]: Oh I think we need to tell them the risks

Comment (ThatNerdyPikachu) [reply common/install.sh]: No, I agree.

Just word it a bit differently.

File: common/install.sh (hunk 1 above)
Comment (thehappydinoa) [reply common/install.sh]: I don't want them to feel bad, it works for some Nexus users and Pixel 2 users

**Review by ThatNerdyPikachu — APPROVED** [review]
If anything comes up grammar-wise, I'll just hotfix to master.

**Pull request merged**